backend/app/tests/crud/collections/collection/test_crud_collection_delete.py

```
@@ -1,17 +1,38 @@
 
 from app.core.config import settings
 from app.crud import CollectionCrud
-from app.models import APIKey
+from app.models import APIKey, Collection
 from app.crud.rag import OpenAIAssistantCrud
 from app.tests.utils.utils import get_project
 from app.tests.utils.document import DocumentStore
 from app.tests.utils.collection import get_collection, uuid_increment
 
 
+def get_collection(db: Session, client=None, project_id: int = None) -> Collection:
```

Ruff (0.14.0)

Redefinition of unused `get_collection` from line 13

(F811)